src/app/code/community/Mageone/Qps/Model/Cron.php

```diff
@@ -68,10 +79,16 @@ public function getRules(): void
                     }
                     $rule->addData($item)->save();
                     $collection->removeItemByKey($rule->getId());
+                    $sendNotification = true;
```

Comment:
Maybe I see it wrong, but currently, we send an email each hour, because $result contains always at least the test rule?

We need to test wether $rule is updated before save.